packages/gatsby/src/state-machines/data-layer/index.ts

```diff
@@ -0,0 +1,95 @@
+import { MachineConfig, Machine } from "xstate"
+import { dataLayerActions } from "./actions"
+import { IDataLayerContext } from "./types"
+import { dataLayerServices } from "./services"
+
+export type DataLayerResult = Pick<
+  IDataLayerContext,
+  | "gatsbyNodeGraphQLFunction"
+  | "graphqlRunner"
+  | "pagesToBuild"
+  | "pagesToDelete"
+>
+
+export const dataLayerStates: MachineConfig<IDataLayerContext, any, any> = {
```

Comment:
NIT - this export doesn't seem imported anywhere and seems to be used only in this module for `dataLayerMachine` - will this be used later on or can we just drop the export?

Reply:
I was exporting it so I could more easily add tests, but I think I'll be moving it out into a separate file first anyway, so I'll rmeove the export.